src/ui/GUIApplication.java

```diff
@@ -52,9 +52,15 @@ public void start(Stage stage) throws Exception {
 	
 	//High Scores screen
 	//@Override
-	public ObservableList<Score> start(Scene highScoresScene) throws Exception {
+	public void start(Scene highScoresScene) throws Exception {
```

Comment:
Het is misschien beter als je deze method `setupHighScoresScene` noemt ipv `start`. De bedoeling van deze method is dat het de `highScoresScene` klaar maakt om getoont te worden.

De `highScoresScene` parameter mag ook weg aangezien we een attribuut met dezeflde naam hebben.

In de `start` method hierboven kan je dan de lijn `highScoresScene = new Scene(layout3)` vervangen door `setupHighScoresScene()`.

PS. het is ook vrij belangrijk dat je je code eens uitprobeert voordat je het commit zodat je kan zien of het wel doet wat het moet doen.